quaterion/train/trainable_model.py

```diff
@@ -345,19 +345,7 @@ def _common_step(self, batch, batch_idx, stage: TrainStage, **kwargs) -> Tensor:
 
         embeddings = self.model(features)
         loss = self.loss(embeddings=embeddings, **targets)
-
-        if (
-            stage == TrainStage.TRAIN
-            and self._enable_xbm
-            and self.trainer.global_step > self._xbm_config.start_iteration
-        ):
-            self._xbm_buffer.queue(embeddings.detach(), targets["groups"].detach())
-
-            memory_embeddings, memory_groups = self._xbm_buffer.get()
-            memory_loss = self.loss(
-                embeddings, targets["groups"], memory_embeddings, memory_groups
-            )
-            loss = loss + self._xbm_config.weight * memory_loss
+        loss = self._maybe_compute_xbm_loss(stage, loss, embeddings, targets)
```

Comment:
Nit: Can we make it just `_compute_xbm_loss` and then `loss += self._compute_xbm_loss()`?